receiver/pluginreceiver/README.md

```diff
@@ -75,6 +75,8 @@ The following keys are used when defining a parameter.
 
 **Warning**: Parameters must adhere to their definition. Invalid parameters will return an error during configuration.
 
+**Warning**: The value given for the timezone parameter type must be a standard UTC timezone, shown in this [timezone list](https://github.com/observIQ/observiq-otel-collector/blob/main/receiver/pluginreceiver/timezone.go).
```

Comment:
Can you actually list this out in a separate markdown file? Sending users to a Go file is not super user friendly.

Something like this:

```markdown
- UTC
- Africa/Abidjan
....
```

Also I think this should be more of a `Note` rather than a `Warning`